crates/wasm_interp/tests/test_opcodes.rs

```diff
@@ -37,11 +37,99 @@ const DEFAULT_PROGRAM_COUNTER: usize = 0;
 // #[test]
 // fn test_brtable() {}
 
-// #[test]
-// fn test_return() {}
+#[test]
+fn test_call_return_no_args() {
```

Comment:
This test is a bit hard to read, I found a better way to do it for the test below. I'll push a commit to refactor this one.